Public/Src/Sandbox/Windows/DetoursServices/DetouredFunctions.cpp

```diff
@@ -1791,6 +2009,40 @@ BOOL WINAPI Detoured_CreateProcessW(
     _In_        LPSTARTUPINFOW        lpStartupInfo,
     _Out_       LPPROCESS_INFORMATION lpProcessInformation)
 {
+    if (g_substituteProcessExecutionShimPath != nullptr)
+    {
+        wstring command;
+        wstring commandArgs;
+        const wchar_t *lpCommandArgs;
+        if (lpApplicationName == nullptr)
+        {
+            FindApplicationNameFromCommandLine(lpCommandLine, command, commandArgs);

```

Comment:
Maybe this is new Cpp but it reads werid to me that the out parameter are not marked with `&`